inference-engine/thirdparty/clDNN/src/impls/ocl/primitive_base.hpp

```diff
@@ -128,6 +108,21 @@ struct typed_primitive_impl_ocl : public typed_primitive_impl<PType> {
         }
     }
 
+    std::vector<layout> get_internal_buffer_layouts_impl() override {
+        if (_kernel_data.internalBufferSizes.empty())
+            return {};
+
+        std::vector<layout> layouts;
+        auto dtype = from_data_type(_kernel_data.internalBufferDataType);
+        const auto bpp = data_type_traits::size_of(dtype);
+        for (auto size : _kernel_data.internalBufferSizes) {
+            layout inbuf_layout = {dtype, format::bfyx, // simple linear format (flattern to x channel)
+                                    {1, 1, 1, (tensor::value_type)(size / bpp)}};
+            layouts.push_back(inbuf_layout);
+        }
+        return std::move(layouts);
```

Comment:
this `std::move` prevents return value optimization by compiler, so I believe it's not needed and you should just use `return layouts;`

Reply:
Fixed